CHANGELOG.md

```diff
@@ -214,6 +218,23 @@ All notable changes to this project will be documented in this file.
 - Fix bugs with upload progress (#34325 by @ChaosExAnima)
 - Fix being unable to hide controls in full screen video in web UI (#34308 by @Gargron)
 - Fix extra space under left-indented vertical videos (#34313 by @ClearlyClaire)
+- Fix missing vertical borders on search input in new mobile menu (light theme) (#35072 by @diondiondion)
```

Comment:
Not sure if we should include this as it was a fix for a feature introduced after the last release?

Reply:
Good catch, the CSS that was removed was far older but was not in effect. Should be folded into the sidebar change.